src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
> it's more like "specification that would yield the same result". 

Ah, I see.  In any case, wildcard doesn't really mean whatever it happened to deliver no? There's the balanceChanges etc in the outcome for that.

I think you could pretty easily have a 1:1 mapping here from the RippleAPI <-> Rippled Payment source/destination amounts. (It would imply disallowing explicit wildcarding)